2025-07-16 14:07:35,116 INFO    MainThread:6267 [wandb_setup.py:_flush():68] Current SDK version is 0.19.10
2025-07-16 14:07:35,116 INFO    MainThread:6267 [wandb_setup.py:_flush():68] Configure stats pid to 6267
2025-07-16 14:07:35,116 INFO    MainThread:6267 [wandb_setup.py:_flush():68] Loading settings from /home/fdittaro/.config/wandb/settings
2025-07-16 14:07:35,116 INFO    MainThread:6267 [wandb_setup.py:_flush():68] Loading settings from /media/HDD/fdittaro/Second_traing_phase/ENB0+UNet/WE3DS/wandb/settings
2025-07-16 14:07:35,116 INFO    MainThread:6267 [wandb_setup.py:_flush():68] Loading settings from environment variables
2025-07-16 14:07:35,116 INFO    MainThread:6267 [wandb_init.py:setup_run_log_directory():724] Logging user logs to /media/HDD/fdittaro/Second_traing_phase/ENB0+UNet/WE3DS/wandb/run-20250716_140735-nck0cw4w/logs/debug.log
2025-07-16 14:07:35,116 INFO    MainThread:6267 [wandb_init.py:setup_run_log_directory():725] Logging internal logs to /media/HDD/fdittaro/Second_traing_phase/ENB0+UNet/WE3DS/wandb/run-20250716_140735-nck0cw4w/logs/debug-internal.log
2025-07-16 14:07:35,116 INFO    MainThread:6267 [wandb_init.py:monkeypatch_ipython():643] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f7d611512b0>
2025-07-16 14:07:35,116 INFO    MainThread:6267 [wandb_init.py:init():852] calling init triggers
2025-07-16 14:07:35,116 INFO    MainThread:6267 [wandb_init.py:init():857] wandb.init called with sweep_config: {}
config: {'epochs': 50, 'batch_size': 64, '_wandb': {}}
2025-07-16 14:07:35,116 INFO    MainThread:6267 [wandb_init.py:init():893] starting backend
2025-07-16 14:07:35,116 INFO    MainThread:6267 [wandb_init.py:init():897] sending inform_init request
2025-07-16 14:07:35,123 INFO    MainThread:6267 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-07-16 14:07:35,123 INFO    MainThread:6267 [wandb_init.py:init():907] backend started and connected
2025-07-16 14:07:35,128 INFO    MainThread:6267 [wandb_run.py:_label_probe_notebook():1369] probe notebook
2025-07-16 14:07:35,128 INFO    MainThread:6267 [wandb_run.py:_label_probe_notebook():1379] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2025-07-16 14:07:35,128 INFO    MainThread:6267 [wandb_init.py:init():1002] updated telemetry
2025-07-16 14:07:35,129 INFO    MainThread:6267 [wandb_init.py:init():1026] communicating run to backend with 90.0 second timeout
2025-07-16 14:07:35,734 INFO    MainThread:6267 [wandb_init.py:init():1101] starting run threads in backend
2025-07-16 14:07:35,860 INFO    MainThread:6267 [wandb_run.py:_console_start():2566] atexit reg
2025-07-16 14:07:35,861 INFO    MainThread:6267 [wandb_run.py:_redirect():2414] redirect: wrap_raw
2025-07-16 14:07:35,861 INFO    MainThread:6267 [wandb_run.py:_redirect():2483] Wrapping output streams.
2025-07-16 14:07:35,861 INFO    MainThread:6267 [wandb_run.py:_redirect():2506] Redirects installed.
2025-07-16 14:07:35,863 INFO    MainThread:6267 [wandb_init.py:init():1147] run started, returning control to user process
2025-07-16 14:07:35,864 INFO    MainThread:6267 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-07-16 14:07:38,979 INFO    MainThread:6267 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-07-16 14:07:38,979 INFO    MainThread:6267 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-07-16 14:07:38,983 INFO    MainThread:6267 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-07-16 14:07:51,291 INFO    MainThread:6267 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-07-16 14:07:51,292 INFO    MainThread:6267 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-07-16 14:07:51,303 INFO    MainThread:6267 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-07-16 14:07:53,452 INFO    MainThread:6267 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-07-16 14:07:53,452 INFO    MainThread:6267 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-07-16 14:07:53,529 INFO    MainThread:6267 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-07-16 14:07:55,671 INFO    MainThread:6267 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-07-16 14:07:55,671 INFO    MainThread:6267 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-07-16 14:07:56,614 INFO    MainThread:6267 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-07-16 14:07:57,419 INFO    MainThread:6267 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-07-16 14:07:57,419 INFO    MainThread:6267 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-07-16 14:07:57,420 INFO    MainThread:6267 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-07-16 14:07:59,531 INFO    MainThread:6267 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-07-16 14:07:59,531 INFO    MainThread:6267 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-07-16 14:07:59,532 INFO    MainThread:6267 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-07-16 14:08:06,440 INFO    MainThread:6267 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-07-16 14:08:06,441 INFO    MainThread:6267 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-07-16 14:13:13,003 INFO    MainThread:6267 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-07-16 14:13:17,924 INFO    MainThread:6267 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-07-16 14:13:17,924 INFO    MainThread:6267 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-07-16 14:14:06,628 INFO    MainThread:6267 [wandb_init.py:_post_run_cell_hook():613] [no run ID] resuming backend
2025-07-16 14:14:35,848 INFO    MainThread:6267 [jupyter.py:save_ipynb():401] [no run ID] not saving jupyter notebook
2025-07-16 14:14:35,849 INFO    MainThread:6267 [wandb_init.py:_pre_run_cell_hook():602] [no run ID] pausing backend
2025-07-16 14:14:35,878 INFO    MainThread:6267 [wandb_run.py:_finish():2314] finishing run federico-dittaro-university-of-udine/Second_phase_WE3DS_dataset-segmentation/nck0cw4w
2025-07-16 14:14:35,878 INFO    MainThread:6267 [jupyter.py:save_history():477] not saving jupyter history
2025-07-16 14:14:35,878 INFO    MainThread:6267 [jupyter.py:save_ipynb():401] not saving jupyter notebook
2025-07-16 14:14:35,879 INFO    MainThread:6267 [wandb_init.py:_jupyter_teardown():628] cleaning up jupyter logic
2025-07-16 14:14:35,879 INFO    MainThread:6267 [wandb_run.py:_atexit_cleanup():2531] got exitcode: 0
2025-07-16 14:14:35,879 INFO    MainThread:6267 [wandb_run.py:_restore():2513] restore
2025-07-16 14:14:35,879 INFO    MainThread:6267 [wandb_run.py:_restore():2519] restore done
2025-07-16 14:14:48,861 WARNING MainThread:6267 [wandb_init.py:init():1654] [no run ID] interrupted
Traceback (most recent call last):
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 1650, in init
    return wi.init(run_settings, run_config, run_printer)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 873, in init
    self._wl.finish_all_active_runs()
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_setup.py", line 150, in finish_all_active_runs
    run.finish()
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 503, in wrapper
    return func(self, *args, **kwargs)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 451, in wrapper
    return func(self, *args, **kwargs)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 2302, in finish
    return self._finish(exit_code)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 2330, in _finish
    self._atexit_cleanup(exit_code=exit_code)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 2543, in _atexit_cleanup
    self._on_finish()
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 2799, in _on_finish
    wait_with_progress(
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/mailbox/wait_with_progress.py", line 24, in wait_with_progress
    return wait_all_with_progress(
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/mailbox/wait_with_progress.py", line 87, in wait_all_with_progress
    return asyncio_compat.run(progress_loop_with_timeout)
  File "/media/HDD/fdittaro/env/lib/python3.8/site-packages/wandb/sdk/lib/asyncio_compat.py", line 30, in run
    return future.result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 434, in result
    self._condition.wait(timeout)
  File "/usr/lib/python3.8/threading.py", line 302, in wait
    waiter.acquire()
KeyboardInterrupt
2025-07-16 14:14:49,901 INFO    MsgRouterThr:6267 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 2 handles.
